### DISCUSSION TITLE
Can we add to faster-whisper an MP3 to WAV Converting argument?

### DISCUSSION THREAD

**Discussion by BanQuanChung** [Purfview/whisper-standalone-win#373] [Q&A] [answered]
Hello @Purfview and everyone!
I find the Standalone Faster-Whisper more accurate when it processes .wav files instead of .mp3.
I tried using ffmpeg to convert mp3 files into wav files.
Can Purfview or anyone show me the way to incorporate this function inside the Whisper or its One-Click-Transtribe.bat so the work requires fewer steps?

Many thanks!

**Comment by Purfview** [comment, marked as answer]
>Can we add to faster-whisper an MP3 to WAV Converting argument?

No need as everything is converted to "wav" already.

>I find the Standalone Faster-Whisper more accurate when it processes .wav files instead of .mp3

You found illusory correlation. 😁

PS
>Illusory correlation is a psychological phenomenon where people perceive a connection between two variables when there is no actual relationship. It can occur when rare or unusual events happen at the same time, and people form a false association based on little evidence.

**Comment by ghost** [comment]
why tho? you are converting lossy to lossless which is making fake lossless. so yeah.
